### ISSUE TITLE
Unnatural villager behavior

### ISSUE THREAD

**Issue by GoldenTotem** [Tropicraft/Tropicraft#411]
![Snipaste_2021-08-29_22-12-47](https://user-images.githubusercontent.com/70516358/131469078-05c2b790-d456-4300-9a10-fe13e21f9099.png)
## Issue Description
When the villagers sit down, they look like they are sitting in the air

Tropicraft Version:   9.0.1
Forge Version:   36.2.2

**Comment by Dragon-Seeker** [comment]
I noticed this with the Fabric Port of tropicraft, Seems to be a issue with the Koa Model. If one of the forge developers see this, the koa model needs the translation code above render method uncommented and to check for if its riding and if its a child(Mapping names may be different):
```java
if (this.riding) {
   if (this.child) {
      ms.translate(0, 0.3, 0);
       } else {
       ms.translate(0, 0.6, 0);
   }
}
```

**Comment by Dragon-Seeker** [comment]
Yet Now the koa's item being held is floating above the sitting koa

**Comment by NotActuallyTerry** [comment]
Appears to be fixed in 1.18 by [a24ae2c](https://github.com/Tropicraft/Tropicraft/commit/a24ae2c934fd38f3838ccfc6882fcf8eeca75d54)
